### PR TITLE
pull metadata for patch release `1.9.5` into `master`

### PR DESCRIPTION
**Pull request by zacharyburnett**
closed previous PR #7476 in favor of this one so as not to clutter the release branch

### CODE REVIEW

**Review by hbushouse — APPROVED** [review]

File: CHANGES.rst [hunk 1]
```diff
@@ -106,6 +106,10 @@ tweakreg
   additional user-provided rotations and scale corrections to an imaging
   WCS of a calibrated image. [#7430]
 
+1.9.5 (2023-03-02)
+==================
+
+- add ``opencv-python`` to ``requirements-sdp.txt``
```
Comment (hbushouse) [inline CHANGES.rst:112]: Whoops, need to reference a PR number here to make the changelog bot happy.

Comment (zacharyburnett) [reply CHANGES.rst:112]: this change doesn't have a PR number, but I can add the label to this PR

Comment (hbushouse) [reply CHANGES.rst:112]: Or just add the "no-changelog-entry-required" label.

Comment (hbushouse) [reply CHANGES.rst:112]: Ah, I see you beat me to it!

**Pull request merged**